test-cases/tests/uninstallation/o03-verify-sendgrid-credentials-are-removed.md

```diff
@@ -0,0 +1,29 @@
+---
+automation:
+  - INTLY-9950
+environments:
+  - osd-fresh-install
+estimate: 30m
+tags:
+  - per-release
+---
+
+# O03 - Verify SendGrid credentials are removed
+
+## Description
+
+This test case should verify that the OCM SendGrid Service has correctly removed the SendGrid sub account for the cluster.
+
+## Prerequisites
+
+- The ID of the OpenShift cluster from OCM e.g. `1f3ufk5dn7suk8m4um8op9k8le9gi64h`
+- SendGrid credentials for the SendGrid account
+- The OpenShift cluster has been deleted
+
+## Steps
+
+1. Login to [SendGrid](https://app.sendgrid.com/login/)
```

Comment:
just a one-step suggestion how we can do this without using UI which would require to only have the Sendgrid token, but I'm fine with that UI option as well :)

```
curl -ks 'https://api.sendgrid.com/v3/subusers' -H 'Authorization: Bearer <SENDGRID_API_KEY>'  | jq -r '.[] | .username' | grep <OPENSHIFT_CLUSTER_ID>
```

Reply:
That's a much better idea, thanks @psturc 👍